src/pages/JournalEntryFormPage/UpdateJournalEntryForm/UpdateJournalEntryForm.tsx

```diff
@@ -56,13 +58,13 @@ const UpdateJournalEntryForm: React.FC<UpdateJournalEntryFormProps> = (props) =>
         if (JSON.stringify(updatedJournalEntry) !== JSON.stringify(previousEntry)) {
             try {
                 updateJournalEntry.mutate(updatedJournalEntry);
-                handleSubmitResults("success");
+                handleSubmitResults(false);
             } catch (error) {
                 console.log(error);
-                handleSubmitResults("error");
+                handleSubmitResults(true);
```

Comment:
This could be moved out of a try/catch and use the onError and onSuccess like in the newJournalEntryForm